server/src/main/java/io/crate/metadata/doc/DocTableInfo.java

```diff
@@ -344,6 +348,7 @@ public Set<Reference> droppedColumns() {
      */
     public TranslogIndexer getTranslogIndexer() {
         if (this.translogIndexer == null) {
+            LOGGER.debug(() -> "Building translog indexer for " + this.ident + " using mapping " + this.mapping().mapping);
```

Comment:
Same here, please surround with `if (LOGGER.isDebugEnabled())` to void expensive operations if not enabled.